webots_ros2_driver/webots_ros2_driver/utils.py

```diff
@@ -105,7 +106,26 @@ def get_wsl_ip_address():
                 return tokens[1]
     finally:
         file.close()
-    
+
+def is_macOS():
```

Comment:
I would replace this with:
```suggestion
def has_shared_folder():
```
as it may not be always specific to macOS.